public/scripts/group-chats.js

```diff
@@ -1309,11 +1330,13 @@ function getGroupCharacterBlock(character) {
     const isFav = character.fav || character.fav == 'true';
     template.data('id', character.avatar);
     template.find('.avatar img').attr({ 'src': avatar, 'title': character.avatar });
-    template.find('.ch_name').text(character.name);
-    template.attr('chid', characters.indexOf(character));
+    template.find('.ch_name').text(character.name + (character.queueOrder > 0?'   (#' + character.queueOrder + ')':''));    template.attr('chid', characters.indexOf(character));
```

Comment:
Would prefer it if there would be another div/span block for the queue number. Makes removing, hiding and styling them easier. And some extensions/or functions sometimes query `.ch_name` to get the display name.

I think this should be easily achievable by modfying the template itself in HTML.

Reply:
Done